src/tools/illink/src/ILLink.RoslynAnalyzer/DataFlow/LocalDataFlowVisitor.cs

```diff
@@ -358,16 +355,7 @@ TValue ProcessSingleTargetAssignment (IOperation targetOperation, IAssignmentOpe
 			// can show up in a flow capture reference (for example, where the right-hand side
 			// is a null-coalescing operator).
 			default:
-				// NoneOperation represents operations which are unimplemented by Roslyn
-				// (don't have specific I*Operation types), such as pointer dereferences.
-				if (targetOperation.Kind is OperationKind.None)
-					break;
-
-				// Assert on anything else as it means we need to implement support for it
-				// but do not throw here as it means new Roslyn version could cause the analyzer to crash
-				// which is not fixable by the user. The analyzer is not going to be 100% correct no matter what we do
-				// so effectively ignoring constructs it doesn't understand is OK.
-				Debug.Fail ($"{targetOperation.GetType ()}: {targetOperation.Syntax.GetLocation ().GetLineSpan ()}");
```

Comment:
Can we use release builds of the analyzer in local development? Such asserts make it really hard to continue working when they occur.